dd-sdk-android-core/src/main/kotlin/com/datadog/android/v2/api/InternalLogger.kt

```diff
@@ -64,12 +64,15 @@ interface InternalLogger {
      * @param target the target handler for the log
      * @param message the log message
      * @param throwable an optional throwable error
+     * @param onlyOnce whether only one instance of the message should be sent per lifetime of the
```

Comment:
should it be per target as well? otherwise if same message has different targets, it will go to the first, but not to the second probably?

EDIT: ok, filters are per target in the implementation. Maybe we should update doc then to make it clear.